plugin/.gitignore

```diff
@@ -1,2 +1 @@
-README.md
```

Comment:
Not sure why this was originally included, but on reverting the reorganization of the tests that I had done in a previous commit, the `README` files weren't being included so this needed to be removed